zonemaster/zonemaster-gui
# Upgrade angular to version 11

## Purpose

Upgrade to angular 11

## Context

\-

## Changes

* Upgarde to angular 11

## How to test this PR

* Run the GUI, check that nothing is broken
* Build a release, it should compile successfully
